apps/dashboard/src/app/(dashboard)/dashboard/contracts/deploy/page.tsx

```diff
@@ -0,0 +1,17 @@
+import { redirect } from "next/navigation";
+import { getAuthTokenWalletAddress } from "../../../../api/lib/getAuthToken";
+import { DeployedContractsPage } from "../../../../team/[team_slug]/[project_slug]/contracts/_components/DeployedContractsPage";
+
+export default function Page() {
+  const accountAddress = getAuthTokenWalletAddress();
+  if (!accountAddress) {
+    return redirect(`/login?next=${encodeURIComponent("/dashboard")}`);
```

Comment:
should be able to redirect back to this page, right? `/dashboard/contracts/deploy`

Reply:
updated